vms/platformvm/txs/executor/staker_tx_verification_test.go

```diff
@@ -99,6 +103,9 @@ func TestVerifyAddPermissionlessValidatorTx(t *testing.T) {
 			backendF: func(*gomock.Controller) *Backend {
 				return &Backend{
 					Ctx: snow.DefaultContextTest(),
+					Config: &config.Config{
+						DurangoTime: time.Time{}, // activate latest fork
```

Comment:
should we specify a time with 0 instead? just for better observability.

Reply:
Yes, rn we don't consistently use time.Time{} vs time.Unix(0,0) in UTs. I guess I can start using `time.Unix(0,0)` for the configs I add in this PR. Will have a pass on a separate PR to update the rest.